content/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request.md

```diff
@@ -9,41 +9,41 @@ versions:
+Anyone with read permissions to a repository can create a pull request, but you must have write permissions to create a branch. If you want to create a new branch for your pull request and don't have write permissions to the repository, you can fork the repository first. The 'fork' function copies the repository to your profile, so that you can make changes to it on your own and submit them for review, without changing the master branch itself. For more information, read "[Creating a pull request from a fork](/articles/creating-a-pull-request-from-a-fork)" and "[About forks](/articles/about-forks)."
 
-You can specify which branch you'd like to merge your changes into when you create your pull request. Pull requests can only be opened between two branches that are different.
+You can specify which branch you'd like to merge your changes into when you create your pull request. Pull requests can only be opened between two different branches.
 
 {% data reusables.pull_requests.perms-to-open-pull-request %}
 
 {% data reusables.pull_requests.close-issues-using-keywords %}
 
 ### Changing the branch range and destination repository
 
-By default, pull requests are based on the parent repository's default branch. For more information, see "[About branches](/github/collaborating-with-issues-and-pull-requests/about-branches#about-the-default-branch)."
+By default, pull requests are based on the parent repository's default branch. For more information, read "[About branches](/github/collaborating-with-issues-and-pull-requests/about-branches#about-the-default-branch)."
 
 If the default parent repository isn't correct, you can change both the parent repository and the branch with the drop-down lists. You can also swap your head and base branches with the drop-down lists to establish diffs between reference points. References here must be branch names in your GitHub repository.
 
 ![Pull Request editing branches](/assets/images/help/pull_requests/pull-request-review-edit-branch.png)
 
 When thinking about branches, remember that the *base branch* is **where** changes should be applied, the *head branch* contains **what** you would like to be applied.
 
-When you change the base repository, you also change notifications for the pull request. Everyone that can push to the base repository will receive an email notification and see the new pull request in their dashboard the next time they sign in.
+When you change the base repository, you also change notifications for the pull request. Everyone that can push to the base repository will receive an email notification and see the new pull request in their dashboard the next time they sign in to GitHub.
```

Comment:
This isn't accurate. It depends on each account's subscription preferences to the base repo.

(Amusingly, I have base repos that I own where I wasn't subscribed which means I missed out on people's PRs since I didn't get notifications.)

I'm not familiar w/ `dashboard`, but I suspect `notifications` is more accurate than dashboard.